src/components/Banner/Banner.tsx

```diff
@@ -117,27 +126,37 @@ const Banner: React.FC<BannerProps> = ({
       </IconPadbox>
       <Padbox paddingSize={SpaceSizes.md}>
         <Inline
-          align="center"
+          align="start"
           gap={SpaceSizes.mdPlus}
           stretch={StretchEnum.start}
         >
-          <Text size={TextSizes.md}>{children}</Text>
-          {isNonEmptyArray(actions) &&
-            actions.map((action) => (
-              <StyledButton
-                key={action.name}
-                color={ButtonColors.primary}
-                href={
-                  (action as AbsoluteLinkActionKind<[React.MouseEvent]>).href
-                }
-                name={action.name}
-                to={(action as RelativeLinkActionKind<[React.MouseEvent]>).to}
-                variant={ButtonVariants.text}
-                onClick={action.onClick}
-              >
-                {action.label}
-              </StyledButton>
-            ))}
+          <Stack gap={SpaceSizes.sm} justify="start">
+            <Text color={theme.colors.neutral[900]} size={TextSizes.md}>
```

Comment:
Please use `as="div"`. Default tag for `Text` component is `span` so if somebody put block element inside `children` it would break validity.